README.md

```diff
@@ -1,3 +1,32 @@
 # StashJS Examples
 
-Example code that demonstrates how to use StashJS.
\ No newline at end of file
+Example code that demonstrates how to use StashJS.
+
+
+## Usage
+
+1. Make sure you've got the necessary dependencies installed:
+
+  ```bash
+  npm install
+  ```
+
+2. Have a CipherStash profile setup:
```

Comment:
Let's link here to [the docs](https://docs.cipherstash.com/tutorials/getting-started/install-stash-cli.html), so people who hit this repo have some context about what the `stash` command is, and how to login.

Reply:
Excellent catch.  Added a commit for that.